packages/flutter_tools/lib/src/build_system/targets/common.dart

```diff
@@ -394,3 +395,45 @@ abstract class CopyFlutterAotBundle extends Target {
     environment.buildDir.childFile('app.so').copySync(outputFile.path);
   }
 }
+
+/// Lipo CLI tool wrapper shared by iOS and macOS builds.
+class Lipo {
+  /// Static only.
+  Lipo._();
+
+  /// Create a "fat" binary by combining multiple architecture-specific ones.
+  static Future<void> create(
+      Environment environment,
+      List<DarwinArch> darwinArchs,
+      String path,
```

Comment:
Maybe `relativePath`? Also would be nice if this and `inputDir` were named parameters so they don't get mixed up.

Reply:
changed